TarkovMapOverlay/App.config

```diff
@@ -1,6 +1,53 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
+    <configSections>
+        <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+            <section name="TarkovMapOverlay.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
```

Comment:
Is it possbile to get rid of the public key token here or do we need it?

Reply:
guess it's needed in some way, i get fileNotFoundExcpetion if i delete that, that's also auto-generated. Don't worry it's not a big deal, that's just an "Identifier Code" for the App itself.